cmd/internal/cli/actions_linux.go

```diff
@@ -216,7 +216,7 @@ func execStarter(cobraCmd *cobra.Command, image string, args []string, name stri
 	currMask := syscall.Umask(0o022)
 	if !NoUmask {
 		// Save the current umask, to be set for the process run in the container
-		// https://github.com/hpcng/singularity/issues/5214
+		// https://github.com/apptainer/apptainer/issues/5214
```

Comment:
These should actually be `apptainer/singularity/issues` since this repo doesn't contain the github issues of the old project. I think a replacement of `apptainer/apptainer/issues` to `apptainer/singularity/issues` will be safe to perform

Reply:
Scratch that, [this](https://github.com/apptainer/apptainer/pull/38/files#diff-c5fe610b0215b144474106251cab954f8af55fe26cbd5d2265fd6ca19109c97bR104) will prevent a nice search and replace :slightly_frowning_face:

Reply:
Maybe just add a slash to the end of the global edit.